mono/eglib/gpath.c

```diff
@@ -251,11 +251,17 @@ g_find_program_in_path (const gchar *program)
 		
 		x = NULL;
 		probe_path = g_build_path (G_DIR_SEPARATOR_S, l, program, NULL);
-		if (access (probe_path, X_OK) == 0){ /* FIXME: on windows this is just a read permissions test */
+#if !defined(NO_HAVE_ACCESS)
+#ifdef G_OS_WIN32
+		if (_access (probe_path, X_OK) == 0){ /* FIXME: on windows this is just a read permissions test */
```

Comment:
We should probably just declare a `g_access` (same way as `g_unlink`) and use that here.

Reply:
I can do that.

Reply:
I've fixed this at 70677aa56309c7635ba748307a0a66bad5b924db.